Sources/App/Views/PackageController/PackageShow+Model.swift

```diff
@@ -176,6 +176,7 @@ extension PackageShow.Model {
         "https://github.com/\(repositoryOwner)/\(repositoryName)"
     }
 
+    // Note: When this gets merged back with the refactored SiteURL, note that it's duplicated in `DocumentationPageProcessor`.
```

Comment:
Probably better than to just make it a static func on `SiteURL` for now. That way we'll address it automatically.

I'll take a look.

Reply:
Never mind, didn't see the additional parameters `repositoryOwner` and `repositoryName` being used.